src/redux/selectors/index.js

```diff
@@ -363,3 +251,118 @@ export const collectionsCountSelector = createSelector(
   [collectionsSelector],
   (collections) => (Object.entries(collections).length),
 );
+
+// eslint-disable-next-line max-len
+const sortMarkedItemsBySubType = ([s1], [s2]) => ((s1.toLowerCase() < s2.toLowerCase()) ? -1 : 1);
+
+const MAX_INTERVAL_COUNT = 50;
+
+export const timelineIntervalsSelector = createSelector(
+  [timelineItemsInRangeSelector, timelineRangeSelector, markedResourcesSelector, resourcesSelector, collectionResourceIdsSelector], // eslint-disable-line max-len
```

Comment:
`collectionResourceIdsSelector` needed to be above ^this b/c `const`/`let` do not hoist